### DISCUSSION TITLE
Does it really take 5 days after SDP approval for the benefits to become active? My class is over in 6 days. :(

### DISCUSSION THREAD

**Discussion by LeFinneus** [community/community#143278] [GitHub Education]
### Hi everyone,

Hello - 
I checked the status of my SDP application and saw that it has been approved. However, I also saw that it could be five days before I can access the benefits (IconScout). Is there any way to expedite this? I have a semester project due on 11/7/24 and very much need access to the icons.

Please advise.

Many thanks! :)

**Comment by davevad93** [comment]
Hi @LeFinneus ,

GitHub no longer provides manual reviews, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @LeFinneus was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
